src/components/structures/ContextualMenu.js

```diff
@@ -41,7 +42,11 @@ function getOrCreateContainer() {
     return container;
 }
 
-export default class ContextualMenu extends React.Component {
+const ARIA_MENU_ITEM_ROLES = new Set(["menuitem", "menuitemcheckbox", "menuitemradio"]);
+// Generic ContextMenu Portal wrapper
+// all options inside the menu should be of role=menuitem/menuitemcheckbox/menuitemradiobutton and have tabIndex={-1}
+// this will allow the ContextMenu to manage its own focus using arrow keys as per the ARIA guidelines.
+export class ContextMenu extends React.Component {
```

Comment:
Hmm, so should this file be renamed as well...? With most of our React components as classes, the file name matches the main class inside, but now we have `ContextualMenu` vs. `ContextMenu`, which is similar but not the same and leaves me wondering while reading the code if both exist as independent things.